pkg/scheduler/apis/config/scheme/scheme.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheme
+
+import (
+	schedulerconfig "github.com/clusternet/clusternet/pkg/scheduler/apis/config"
+	schedulerconfigv1alpha1 "github.com/clusternet/clusternet/pkg/scheduler/apis/config/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+var (
+	// Scheme is the runtime.Scheme to which all scheduler api types are registered.
+	Scheme = runtime.NewScheme()
```

Comment:
We may not need to register this scheme. It is neither for a REST api, nor for a CRD.

Reply:
This is to parse a configuration file like the following:
```
apiVersion: scheduler.config.clusternet.io/v1alpha1
kind: SchedulerConfiguration
profiles:
  -  plugins:
      predict:
        enabled:
        - name: "testPlugin"
        disabled:
        - name: "*"
      assign:
        enabled:
        - name: 'testPlugin'
        disabled:
        - name: '*'
      bind:
        enabled:
        - name: 'testPlugin'
        disabled:
        - name: '*'
```

like kube-scheduler: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/apis/config/scheme/scheme.go

Reply:
It works with `json.Unmarshal`, since we know the exact scheme for deserialziation.